myfitnesspal/client.py

```diff
@@ -220,6 +232,55 @@ def get_date(self, *args, **kwargs):
 
         return day
 
+    def get_measurements(self, measurement='Weight', start=None, end=None):
+        if start is None:
```

Comment:
Ahh; clever!  Using generic dates sounds like a great idea; that way, no matter what way their mental model is set up, it'll work as they're expecting it to.